kernel/src/mm/mod.rs

```diff
@@ -246,6 +255,7 @@ pub fn map_address_space(
     let page_table = crate::PagingImpl::new(mm);
     let page_size = mm.page_size();
 
+    // ???: aren't we mapping the same stuff here as in pmm_pages.for_each...
```

Comment:
Here we are creating an empty page table with invalid entry for all the DRAM. Later on in `pmm_pages.for_each...` we are mapping with correct entries the pages corresponding to the metadata and allocated pages by the PMM.